test/Test/Main.purs

```diff
@@ -3,16 +3,52 @@ module Test.Main where
 import Prelude
 
 import Data.Maybe (Maybe(..), fromMaybe)
-import Data.Number (nan, isNaN, infinity, isFinite, fromString)
+import Effect (Effect)
+import Effect.Console (log)
+
+import Data.Number (isFinite, infinity,
+                    nan, isNaN, fromString)
 import Data.Number.Format (precision, fixed, exponential, toStringWith,
                            toString)
 import Data.Number.Approximate (Fraction(..), Tolerance(..), eqRelative,
                                 eqAbsolute, (≅), (≇))
```

Comment:
It doesn't really matter, but I don't think we format imports like this anywhere else and for consistency's sake it might be nice to format them on one line.

The same is true of these non-ASCII operators from `Data.Number.Approximate` -- but I suppose we're keeping those due to them being a breaking change to remove?

Either way it's no reason to block this from merging, but I wanted to draw attention here.

Reply:
I kept those as is to help show that the port didn't add/remove any code. 

Other than that, I agree that the imports should be reduced to one line again. I'll add a commit that does that.

Reply:
Yeah, I would usually prefer not to introduce non-ASCII operators in the core libraries, but causing a breaking change by removing them is definitely worse in my book.

Reply:
Hm... Yeah, I'm not sure about the non-ASCII syntax.